README.md

```diff
@@ -1,12 +1,15 @@
 # Mozilla Code Review
 
-This project has 2 parts:
+The **Code Review Bot** aims to give early feedback to Mozilla developers about their patches. We automate code analyzers and publish detected issues on Phabricator as soon as possible, and for all revisions.
 
-* `bot` is a Python script running as a Taskcluster task, reporting issues found in analyzer tasks,
-* `frontend` is an administration frontend (in Vue.js) displaying detailed information about current analyses.
+This project has 5 parts:
 
-The analyzer tasks are triggered by [pulselistener](https://github.com/mozilla/release-services/tree/master/src/pulselistener), from the release-services project.
+* `bot` is a Python 3 script running as a Taskcluster task, reporting issues found in analyzer tasks,
```

Comment:
Nit: I wouldn't mention the Python version here and in the other places, it's a detail in the end